app/com/linkedin/drelephant/tuning/hbt/MRConstant.java

```diff
@@ -0,0 +1,55 @@
+package com.linkedin.drelephant.tuning.hbt;
+
+final class MRConstant {
+  static final String MAPPER_TIME_HEURISTIC = "Mapper Time";
```

Comment:
These values are defined in configuration. Either this should be used from configuration or use heuristics class name.